api/sixtyfps-node/native/lib.rs

```diff
@@ -258,15 +262,30 @@ fn to_js_value<'cx>(
         Value::Array(a) => {
             let js_array = JsArray::new(cx, a.len() as _);
             for (i, e) in a.into_iter().enumerate() {
-                let v = to_js_value(e, cx)?;
+                let v = to_js_value(e, cx, persistent_context)?;
                 js_array.set(cx, i as u32, v)?;
             }
             js_array.as_value(cx)
         }
+        Value::Model(model) => {
+            if let Some(js_model) = model.as_any().downcast_ref::<js_model::JsModel>() {
+                js_model.get_object(cx, persistent_context)?.as_value(cx)
+            } else {
+                // TODO: this should probably create a proxy object instead of extracting the entire model. On the other hand
+                // we should encounter this only if the model was created in .60, which is when it'll be an array
+                // of values.
```

Comment:
But this means that the model from .60 are different.
If you have a model set from .js, you can do
```js
instance.the_model[idx] = something;
```
and that will update the UI.

But if the model is comming from .60 and so this code path is run, then one will be operating on a copy of the model instead.
So we are back to the inconsistancy.

Reply:
Yes, that's a good point. Previously this code caused a panic, now it will silently "fail" (not update the UI).